src/EnergyPlus/WeatherManager.cc

```diff
@@ -4324,12 +4324,14 @@ namespace WeatherManager {
                 flags.ADVANCE("No");
                 ObjexxFCL::gio::write(OutputFileInits, fmtA, flags) << StringOut;
             }
+
+            // Hum Ind Type, Hum Ind Value at Max Temp
             if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_WetBulb) {
                 StringOut = "Wetbulb," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {C},";
             } else if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_DewPoint) {
                 StringOut = "Dewpoint," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {C},";
             } else if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_Enthalpy) {
-                StringOut = "Enthalpy," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {kJ/kg},";
+                StringOut = "Enthalpy," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 2) + " {J/kg},";
             } else if (DesDayInput(EnvrnNum).HumIndType == DDHumIndType_HumRatio) {
                 StringOut = "HumidityRatio," + RoundSigDigits(DesDayInput(EnvrnNum).HumIndValue, 4) + " {},";
```

Comment:
Units for this should be kgWater/kgDryAir (per IDD)

Reply:
Done, tested that it outputs `HumidityRatio,8.0000E-003,{kgWater/kgDryAir}`